### ISSUE TITLE
Missing Brand Icons and Outline Icons

### ISSUE THREAD

**Issue by erecord** [Semantic-Org/UI-Icon#11]
When installing this package through npm `npm i semantic-ui-icon`, the assets/fonts/ folder does not contain the Brand Icons and Outline Icons.

This means when `npm run dev` I get errors about these Icon files not being found. For now, my fix has been to delete the Brand Icons and Outline Icons section, within the icon.css.

**Comment by tigran-cl** [comment]
Same here...

`ERROR in ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css
Module not found: Error: Can't resolve './assets/fonts/brand-icons.eot' in 
 @ ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css 7:64994-65035 7:65061-65102
 @ ./node_modules/semantic-ui-icon/icon.min.css
 @ ./resources/assets/js/app.js
 @ multi ./resources/assets/js/app.js ./resources/assets/sass/app.scss

ERROR in ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css
Module not found: Error: Can't resolve './assets/fonts/brand-icons.svg' in 
 @ ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css 7:65401-65442
 @ ./node_modules/semantic-ui-icon/icon.min.css
 @ ./resources/assets/js/app.js
 @ multi ./resources/assets/js/app.js ./resources/assets/sass/app.scss

ERROR in ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css
Module not found: Error: Can't resolve './assets/fonts/brand-icons.ttf' in 
 @ ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css 7:65319-65360
 @ ./node_modules/semantic-ui-icon/icon.min.css
 @ ./resources/assets/js/app.js
 @ multi ./resources/assets/js/app.js ./resources/assets/sass/app.scss

ERROR in ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css
Module not found: Error: Can't resolve './assets/fonts/brand-icons.woff' in 
 @ ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css 7:65240-65282
 @ ./node_modules/semantic-ui-icon/icon.min.css
 @ ./resources/assets/js/app.js
 @ multi ./resources/assets/js/app.js ./resources/assets/sass/app.scss

ERROR in ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css
Module not found: Error: Can't resolve './assets/fonts/brand-icons.woff2' in 
 @ ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css 7:65159-65202
 @ ./node_modules/semantic-ui-icon/icon.min.css
 @ ./resources/assets/js/app.js
 @ multi ./resources/assets/js/app.js ./resources/assets/sass/app.scss

ERROR in ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css
Module not found: Error: Can't resolve './assets/fonts/outline-icons.eot' in  @ ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css 7:58029-58072 7:58098-58141
 @ ./node_modules/semantic-ui-icon/icon.min.css
 @ ./resources/assets/js/app.js
 @ multi ./resources/assets/js/app.js ./resources/assets/sass/app.scss

ERROR in ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css
Module not found: Error: Can't resolve './assets/fonts/outline-icons.svg' in 
 @ ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css 7:58446-58489
 @ ./node_modules/semantic-ui-icon/icon.min.css
 @ ./resources/assets/js/app.js
 @ multi ./resources/assets/js/app.js ./resources/assets/sass/app.scss

ERROR in ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css
Module not found: Error: Can't resolve './assets/fonts/outline-icons.ttf' in 
 @ ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css 7:58362-58405
 @ ./node_modules/semantic-ui-icon/icon.min.css
 @ ./resources/assets/js/app.js
 @ multi ./resources/assets/js/app.js ./resources/assets/sass/app.scss

ERROR in ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css
Module not found: Error: Can't resolve './assets/fonts/outline-icons.woff' in 
 @ ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css 7:58281-58325
 @ ./node_modules/semantic-ui-icon/icon.min.css
 @ ./resources/assets/js/app.js
 @ multi ./resources/assets/js/app.js ./resources/assets/sass/app.scss

ERROR in ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css
Module not found: Error: Can't resolve './assets/fonts/outline-icons.woff2' in 
 @ ./node_modules/css-loader!./node_modules/semantic-ui-icon/icon.min.css 7:58198-58243
 @ ./node_modules/semantic-ui-icon/icon.min.css
 @ ./resources/assets/js/app.js
 @ multi ./resources/assets/js/app.js ./resources/assets/sass/app.scss`

**Comment by gitney1** [comment]
> When installing this package through npm `npm i semantic-ui-icon`, the assets/fonts/ folder does not contain the Brand Icons and Outline Icons.
> 
> This means when `npm run dev` I get errors about these Icon files not being found. For now, my fix has been to delete the Brand Icons and Outline Icons section, within the icon.css.

actually the font files in this folder contains it already, so just remove all "outline-" that contains your icon.min.css and that worked for me. (I didn't test yet for brand icons and for the not minified css file, but I guess it's same)

**Comment by JelliottR** [comment]
> > When installing this package through npm `npm i semantic-ui-icon`, the assets/fonts/ folder does not contain the Brand Icons and Outline Icons.
> > This means when `npm run dev` I get errors about these Icon files not being found. For now, my fix has been to delete the Brand Icons and Outline Icons section, within the icon.css.
> 
> actually the font files in this folder contains it already, so just remove all "outline-" that contains your icon.min.css and that worked for me. (I didn't test yet for brand icons and for the not minified css file, but I guess it's same)

Thanks! I haven't used this package in a while, but will in the future, and I'm sure your reply will help others

**Comment by gitney1** [comment]
> > > When installing this package through npm `npm i semantic-ui-icon`, the assets/fonts/ folder does not contain the Brand Icons and Outline Icons.
> > > This means when `npm run dev` I get errors about these Icon files not being found. For now, my fix has been to delete the Brand Icons and Outline Icons section, within the icon.css.
> > 
> > 
> > actually the font files in this folder contains it already, so just remove all "outline-" that contains your icon.min.css and that worked for me. (I didn't test yet for brand icons and for the not minified css file, but I guess it's same)
> 
> Thanks! I haven't used this package in a while, but will in the future, and I'm sure your reply will help others

Yep thanks :) but I was actually wrong, I had forgotten some other cached library in my project which already used it and were in fact overrided by this one.. So finally, I simply change this "fonts" folder for the full one from Semantic master theme and in "default/assets/fonts" which contains all desired fonts..

**Comment by Eliaxs1900** [comment]
# Important

Please see your [pull request](https://github.com/Semantic-Org/UI-Icon/pulls) @kim366 has post a [commint](https://github.com/Semantic-Org/UI-Icon/pull/10/commits/aa765f6ee071aca330b0f116de4bd966ecdc231d) where are the **all missing fonts** 👍
